scripts/s3manage.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env python
```

Comment:
ok so after a `source .../bin/activate`
You can do `scripts/s3manage.py {command}`

In makefile you'r always using ${PYTHON_CMD} I'm not sure which one is used in that case...